flink-kubernetes/src/main/java/org/apache/flink/kubernetes/kubeclient/KubeClientFactory.java

```diff
@@ -42,18 +42,23 @@ public static FlinkKubeClient fromConfiguration(Configuration flinkConfig) {
 
 		final Config config;
 
+		final String kubeContext = flinkConfig.getString(KubernetesConfigOptions.CONTEXT);
+		if (kubeContext != null) {
+			LOG.info("Configuring K8S client using context {}.", kubeContext);
```

Comment:
I agree to use "Kubernetes" in logging & document as @wangyang0918 suggested.

Reply:
Done